images-dist/modules/com.redhat.osl.bundleinstall/bundle/manifests/logic-operator-rhel9.clusterserviceversion.yaml

```diff
@@ -732,20 +732,20 @@ spec:
                     fieldRef:
                       fieldPath: metadata.namespace
                 - name: RELATED_IMAGE_JOBS_SERVICE_POSTGRESQL
-                  value: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel9:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel9:1.37.0
                 - name: RELATED_IMAGE_JOBS_SERVICE_EPHEMERAL
-                  value: registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel9:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel9:1.37.0
                 - name: RELATED_IMAGE_DATA_INDEX_POSTGRESQL
-                  value: registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel9:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel9:1.37.0
                 - name: RELATED_IMAGE_DATA_INDEX_EPHEMERAL
-                  value: registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel9:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel9:1.37.0
                 - name: RELATED_IMAGE_DB_MIGRATOR_TOOL
-                  value: registry.redhat.io/openshift-serverless-1/logic-db-migrator-tool-rhel9:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-db-migrator-tool-rhel9:1.37.0
                 - name: RELATED_IMAGE_BASE_BUILDER
-                  value: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel9:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel9:1.37.0
                 - name: RELATED_IMAGE_DEVMODE
-                  value: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel9:main
-                image: registry.redhat.io/openshift-serverless-1/logic-rhel9-operator:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel9:1.37.0
+                image: registry.redhat.io/openshift-serverless-1/logic-rhel9-operator:sha256:e7880de7f10215987bd657079f9fb3c10213bf5051fe76ee17c3d10cb0496e8b
```

Comment:
This neeeds to be done manually or automatically in the gitlab sources after the operator is done.

Reply:
there is  https://issues.redhat.com/browse/SRVLOGIC-339 which was a result why we still need to use sha256 sum, but I would like to pave the solution to remove that need and use buildTag or report to RH team to fix it or do the replacement on the fly in CPaaS or by some logic to get that sha256sum automatically